### PR TITLE
02 webpack

### PR DESCRIPTION
**Pull request by navitskyd**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
-import React, { StrictMode } from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
+//import _ from 'lodash';
```
Comment (Vladlan) [inline src/index.js]: You should not have commented code in your repo

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        },
+        resolve: {
+            modules: [APP_DIR,'node_modules'],
+            extensions: ['.js', '.jsx', '.ts', '.tsx', '.css', '.less']
+        },
+        module: {
+            rules: [
+                {
+                    // for any file with a suffix of js or jsx
+                    test: /\.jsx?$/,
```
Comment (Vladlan) [inline webpack.config.js:38]: So are you sure you will not have js fies?

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                },
+                {
+                    test: /\.css$/,
+                    use: [MiniCssExtractPlugin.loader, "style-loader", "css-loader"]
+                }, {
+                    test: /\.less$/,
+                    use: [MiniCssExtractPlugin.loader, "style-loader", "css-loader", "less-loader"]
+                },
+                {
+                    test: /\.tsx?$/,
```
Comment (Vladlan) [inline webpack.config.js:52]: So are you sure you will not have ts fies?

File: package.json [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
+  "name": "movie-FE",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "webpack-dev-server --open",
```
Comment (Vladlan) [inline package.json]: Is this command working?

**Review by Vladlan — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (navitskyd) [reply webpack.config.js:38]: the regex allows to use js and jsx

File: webpack.config.js (hunk 3 above)
Comment (navitskyd) [reply webpack.config.js:52]: ts is also allowed

File: webpack.config.js (hunk 2 above)
Comment (Vladlan) [reply webpack.config.js:38]: agree, my bad

**Review by Vladlan — COMMENTED** [review]
You can abandon or merge this PR. I will add some comments related to webpack in 03-components

File: package.json (hunk 4 above)
Comment (navitskyd) [reply package.json]: deleted

**Pull request merged**